include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -147,7 +148,8 @@ class MjbotsControlLoop : public AbstractRealtimeObject {
   std::string logging_channel_name_;      /// Channel name to publish logs to, leave empty if not publishing
   lcm::LCM lcm_;                          /// LCM object
   LogClass log_data_;                     /// object containing log data
-  LcmSubscriber<InputClass> lcm_sub_;     /// LCM subscriber object
+  LcmSubscriber lcm_sub_;                 /// LCM subscriber object
+  LcmMessageHandler<InputClass> input_sub_;    /// LCM input subscription
```

Comment:
I much prefer this breakout

Still weird we dont have a similar pattern for the log, even though it makes sense mechanically.
This is good. 
Was worried naming here was a little confusing since both of these are kodlab classes, but noticed they are in kodlab namespace, so should be ok.

Reply:
Agreed, I couldn't come to terms with `LcmSubscriber` holding bot the LCM object and the input subscription.